### DISCUSSION TITLE
Relative line number navigation not working properly

### DISCUSSION THREAD

**Discussion by adriangalilea** [helix-editor/helix#9976] [General]
### Summary

When I do `number` + `j`/`k` it doesn't behave as expected.

### Reproduction Steps

<img width="1433" alt="image" src="https://github.com/helix-editor/helix/assets/90320947/7724f9f9-f23e-40d5-a47d-72bc5ea78475">
Pay attention at the left window.

I type 35 + k to arrive at the line `return socialNetworks`
<img width="1440" alt="image" src="https://github.com/helix-editor/helix/assets/90320947/4f043c9e-a54c-40b4-8bcb-40778162619f">
Look where I land tho.

<img width="1440" alt="image" src="https://github.com/helix-editor/helix/assets/90320947/6fd51ffc-127e-4999-90da-0b639ff92ebe">

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

macOS

### Terminal Emulator

Kitty

### Installation Method

brew

### Helix Version

23.10

**Comment by kirawi** [comment]
https://github.com/helix-editor/helix/wiki/FAQ#can-the-jk-bindings-be-changed-to-ignore-soft-wrapping-when-using-a-count-like-3j

**Reply by adriangalilea** [reply]
Hello,

I'm not sure I understand the docs, I get using `3gj` insteado of `3j` altho I'd love if I could change it

Or this code that I'm not sure how it's meant to be used:
```
(if (!= count 0) (move_line_up count) (move_vertical_line_up 0))
```

Thank you!
